crates/services/gas_price_service/src/fuel_gas_price_updater.rs

```diff
@@ -113,16 +117,22 @@ pub trait L2BlockSource: Send + Sync {
     async fn get_l2_block(&mut self, height: BlockHeight) -> Result<BlockInfo>;
 }
 
-#[derive(Debug, Default)]
-pub struct DaCommitDetails {
+#[derive(Debug, Default, Clone)]
+pub struct DaGasPrice {
     pub l2_block_range: core::ops::Range<u32>,
     pub blob_size_bytes: u32,
     pub blob_cost_wei: u32,
-    pub partial_block_heights: Option<[u32; 2]>,
 }
 
-pub trait DaCommitSource: Send + Sync {
-    fn get_da_commit_details(&mut self) -> Result<Option<DaCommitDetails>>;
+pub trait GetDaGasPriceFromSink: Send + Sync + Default + Clone {
```

Comment:
This name leaks implementation details. Doesn't `Sink` imply something like `Arc<Mutex<_>>`. Either way, `GetDaBlockCosts` would be plenty context, don't you think?

Reply:
addressed in https://github.com/FuelLabs/fuel-core/pull/2163/commits/74fc9bf4e08cb0d4a1a5def23f31d2c4fbd31c9f#diff-a87f94b204ea2e176921aba2703aeedb00fad9254c2157038d735d9d53dc889dR95-R98